src/components/_tests_/Alert.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { shallow } from "enzyme";
+import { createElement } from "react";
+
+import { Alert, AlertProps } from "../Alert";
+
+describe("Alert", () => {
+    const message = "This is an error";
+    const renderAlert = (props: AlertProps) => shallow(createElement(Alert, props));
+    let defaultProps: AlertProps;
+
+    beforeEach(() => {
+        defaultProps = {
+            bootstrapStyle: "danger",
+            className: "widget-drop-down-filter",
+            message
+        };
+    });
+
+    it("renders the structure when an alert message is specified", () => {
+        const alert = renderAlert(defaultProps);
+
+        expect(alert).toBeElement(
+            createElement("div", { className: "alert alert-danger widget-drop-down-filter" }, message)
```

Comment:
Change className to your specific widget

Reply:
done